packs/feats/shining-oath.json

```
@@ -31,7 +28,45 @@
             "remaster": false,
             "title": "Pathfinder Core Rulebook"
         },
-        "rules": [],
+        "rules": [
+            {
+                "domain": "damage",
+                "key": "RollOption",
+                "label": "Retributive Strike against an undead",
+                "option": "shining-oath",
+                "predicate": [
+                    "feature:paladin"
+                ],
+                "toggleable": true
+            },
+            {
+                "key": "FlatModifier",
+                "label": "Shining Oath",
+                "predicate": [
+                    "shining-oath",
+                    "target:mode:undead"
+                ],
+                "selector": "strike-damage",
+                "slug": "shining-oath-damage",
+                "type": "circumstance",
+                "value": 4
+            },
+            {
+                "key": "AdjustModifier",
+                "mode": "add",
+                "predicate": [
+                    {
+                        "or": [
+                            "item:proficiency:rank:3",
+                            "item:proficiency:rank:4"
+                        ]
+                    }
```

```suggestion
                    {
                        "gte": [
                            "item:proficiency:rank",
                            3
                        ]
                    }
```
Protects against some horrible future where Paizo drops level 20+ play on us all.